### PR TITLE
Improve Windows/Hyper-V Admin-Password Feature

### PR DESCRIPTION
**Pull request by thutterer**
This is a follow-up on #406.
As discussed there, the attribute gets renamed to `admin_password` and put into a mapping called `windows` to clarify it's function.
In the Crowbar UI a hint gets added to explain it's functionality.

### CODE REVIEW

File: chef/data_bags/crowbar/migrate/provisioner/023_add_hyperv-adminpassword.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 def upgrade ta, td, a, d
-  a['admin_pass'] = ta['admin_pass']
+  a['windows'] = {}
+  a['windows']['admin_password'] = ta['admin_pass']
```
Comment (vuntz) [inline chef/data_bags/crowbar/migrate/provisioner/023_add_hyperv-adminpassword.rb]: You can simplify with `a['windows'] = ta['windows']`, but it's no big deal :-)

**Comment by vuntz** [comment]
+1.

Note for others: I'm not concerned about changing the old migration instead of creating a new one as this migration just got in and really, it won't impact anyone.

File: chef/data_bags/crowbar/migrate/provisioner/023_add_hyperv-adminpassword.rb (hunk 1 above)
Comment (thutterer) [inline chef/data_bags/crowbar/migrate/provisioner/023_add_hyperv-adminpassword.rb]: Updated it to have your line in the migration file.

**Comment by nkrinner** [comment]
+1

**Pull request merged**